single_tech_samples/databricks/sample3_cluster_provisioning_and_data_access/access-data-mount-via-account-key.ipy

```diff
@@ -0,0 +1,27 @@
+# Databricks notebook source
```

Comment:
It id recommended that we follow the same templates for Readmes. You can refer to the [Sample 1 readme ](https://github.com/Azure-Samples/modern-data-warehouse-dataops/blob/single-tech/databricks/single_tech_samples/databricks/sample1_basic_azure_databricks_environment/README.md) as a template

Reply:
Made changes